pkgs/tools/misc/empty/default.nix

```diff
@@ -0,0 +1,49 @@
+{ fetchgit, lib, stdenv, which }:
+
+let
+  baseName = "empty";
+  rev = "6e81f477dba6c2f9086da70b02abec71b64fb16e";
+  baseVersion = "0.6.21b";
+  version = baseVersion + "+1-git-" + (builtins.substring 0 7 rev);
+  name = baseName + "-" + version;
+in stdenv.mkDerivation {
+  inherit name;
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://git.code.sf.net/p/${name}/code";
```

Comment:
```suggestion
    url = "https://git.code.sf.net/p/${pname}-${version}/code";
```

Reply:
This won't work - the repo is based on the name of the project (empty), and there's no version in the path.